### ISSUE TITLE
If main python version is python3, installation raises an error.

### ISSUE THREAD

**Issue by huseyinyilmaz** [Oefenweb/ansible-supervisor#39]
I have python2 and 3 installed on the same machine and I am getting following error:
```
fatal: [104.131.157.212]: FAILED! => {"changed": false, "cmd": "/usr/local/bin/pip3 install supervisor", "failed": true, "msg": "stdout: Collecting supervisor\n  Using cached supervisor-3.3.1.tar.gz\n    Complete output from command python setup.py egg_info:\n    Supervisor requires Python 2.4 or later but does not work on any version of Python 3.  You are using version 3.6.0 (default, Jan 13 2017, 20:56:47)\n    [GCC 5.4.0 20160609].  Please install using a supported version.\n    \n    ----------------------------------------\n\n:stderr: Command \"python setup.py egg_info\" failed with error code 1 in /tmp/pip-build-0bp95_dc/supervisor/\n"}
```

I believe solution is to force python2 pip to be used like this:
https://github.com/ScorpionResponse/ansible-supervisord/blob/0a03f94af6a9ceb966b69690f04af9fa5ef623d6/tasks/main.yml#L18

**Comment by tersmitten** [comment]
Nice catch! Can you make a pull request?

**Comment by huseyinyilmaz** [comment]
Sure.

https://github.com/Oefenweb/ansible-supervisor/pull/40

**Comment by tersmitten** [comment]
Fixed in #40
